backend/src/resolvers/shout.js

```diff
@@ -0,0 +1,51 @@
+export default {
+  Mutation: {
+    shout: async (_object, params, context, _resolveInfo) => {
+      const { id, type } = params
+
+      const session = context.driver.session()
+      let sessionRes = await session.run(
```

Comment:
I don't love the name `sessionRes`
I feel like we should be naming it what is the actual result, which would tempt me to name it `isShouted`, but then I see on line 19, we seem to get a list of records back that we then need to grab `isShouted` from.

I was having a read here https://neo4j.com/docs/driver-manual/current/sessions-transactions/, and it seems the result is from the transaction, not the session. Maybe it would be better `transactionRes`, but maybe I'm just being pedantic, and can be ignored.

Reply:
I am really happy to rename `sessionRes`  to ´transactionRes´ @mattwr18 . That is a very good clarification we can use in the future for all customised mutations. 🤗

I am happy especially if there is nothing else to mention from your side. 😁